nstack-translations-generator/Classes/TranslationsGenerator/GeneratorSettings.swift

```diff
@@ -13,6 +13,7 @@ struct GeneratorSettings {
     var keys: (appID: String, appKey: String)?
     var outputPath: String?
     var flatTranslations: Bool
+    var availableFromObjC: Bool
 }
 
 extension GeneratorSettings {
```

Comment:
Please change the argument.count check so that the maximum number of arguments is 10, as we added a new argument :)